steps/bitrise-step-appdome-build-2secure-ios/1.0.2/step.yml

```
@@ -0,0 +1,48 @@
+title: Appdome-Build-2Secure step for iOS
+summary: |
+  Builds an iOS mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API. For detailes see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
```

Note: the URL is not available (yet)
```suggestion
  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API. For details see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
```